sum=0
def fun_sum(n):
    sum=n+(n-1)
    return sum
try:
    fun_sum(0)
    print(su)

except NameError:
    print("Encountered name error")
except:
    print("Encountered some error")
